test/hocr-wordfreq/hocr-wordfreq.tsht

```diff
@@ -9,5 +9,5 @@ casesensitive=$(hocr-wordfreq -i $TESTDATA/sample.html | head -n 1)
 exec_ok "hocr-wordfreq" "$TESTDATA/sample.html"
 exec_ok "hocr-wordfreq" "-i" "-n 30" "$TESTDATA/sample.html"
 
-match "24\s*the" "$normal" "check that no errors are found when comparing the sample file with itself"
-match "23\s*the" "$casesensitive" "check that no errors are found when comparing the sample file with itself"
+match "24\s*the" "$normal" "check the most frequent word in sample.html"
```

Comment:
Okay, I will delete these changes here and we can continue discussion of these in #100